### PR TITLE
T7426: Install CountDownClock

### PR DESCRIPTION
**Pull request by R4356th**
Per request in T7426.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1454,3 +1454,8 @@
 	branch = REL1_35
 	ignore = dirty
```
Comment (Universal-Omega) [inline .gitmodules:1456]: Remove newline here

File: .gitmodules [hunk 2]
```diff
@@ -1454,3 +1454,8 @@
...
 
+[submodule "extensions/CountDownClock"]
+	path = extensions/CountDownClock
+	url = https://github.com/wikimedia/mediawiki-extensions-CountDownClock
+	branch = REL1_35
+	ignore=dirty
```
Comment (Universal-Omega) [inline .gitmodules:1461]: Add spacing (ignore = dirty)

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (R4356th) [reply .gitmodules:1456]: That's.. unrelated?

File: .gitmodules (hunk 2 above)
Comment (R4356th) [reply .gitmodules]: Done.

File: .gitmodules (hunk 1 above)
Comment (Universal-Omega) [reply .gitmodules:1456]: Oh, but still do you mind removing it? It'S not there anywhere else.

Comment (Universal-Omega) [reply .gitmodules:1456]: Oh but still do you mind removing it? It's not there anywhere else.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**